### DISCUSSION TITLE
Why isn't the `build:webpack` target published on npm?

### DISCUSSION THREAD

**Discussion by stackmystack** [eslint/eslint#17730] [General] [answered]
Hello,

I noticed that the published npm package does not contain the output from `npm run build:webpack` which produces a version of eslint suitable for the web/running in the browser.

I am in that situation, and I would like to access the built script without having to run `npm` myself for technical reasons (please, this is a hard requirement for me).

Is there any reason for it to be excluded?

**Comment by nzakas** [comment, marked as answer]
We do not officially support bundling ESLint for the browser. We can't guarantee that it will work as expected, so we don't publish it.
